packages/react-styles/src/css/components/Topology/topology-controlbar.css

```diff
@@ -11,6 +11,9 @@
   border-radius: var(--pf-global--BorderRadius--sm);
   box-shadow: var(--pf-global--BoxShadow--sm);
 }
+.pf-topology-control-bar__button.pf-c-button.pf-m-tertiary.pf-m-disabled {
```

Comment:
Shouldn't the Button CSS rules be included when you use the `<button>` component from the react code? 

ITW there is a pf-c-button.pf-m-disabled rules in the button.css which theoretically is imported automatically when you use the button.

Reply:
The rule above for `pf-topology-control-bar__button.pf-c-button.pf-m-tertiary` overrides the `. pf-c-button.pf-m-disabled` rule (more specificity). Added this rule to override that rule.

Reply:
Oh got it.. Ideally I would nest the `background-color` rule above, in a :not(.pf-m-disabled) block. In this way you just allow the pf-m-disabled to be applied, instead of overwriting *2* times. 

Anyway, I will approve, and leave it up to you to decide.

Reply:
Agreed, and updated. Thanks!